src/database/index.tsx

```diff
@@ -118,53 +113,6 @@ const buildUpdatePayload = (entity: any) => {
   return payload;
 };
 
-const getInputModelsDocRef = async () => {
```

Comment:
These were the methods that were used to set up the original persistence model.  They served as great guides, but aren't otherwise used so I'm getting rid of them